### ISSUE TITLE
Error just after install

### ISSUE THREAD

**Issue by JulesDevStack** [noftaly/nipinit#1]
Just after install with
`npm i -g nipinit`

I ran nipinit and this error appear : 

```
import fsSync, { promises as fs } from 'fs';
       ^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

**Comment by noftaly** [comment]
May you provide your NodeJS version please? `node -v`

**Comment by noftaly** [comment]
Solved via discord: the Node version was not satisfying the required one: `>=14.8.0`

BUT!
With the big refactor I've just done (available on branch `typescript`), the required Node version is now `^10.17.0 || >=12.0.0`! It will be available in the next `nipinit` release: 2.0.0.

Thanks for your issue and sorry for the inconvenience, I'll close it when the next release is published 🙂

**Comment by noftaly** [comment]
nipinit `v2.0.0` has just been released and supports NodeJS v10.17.0, this issue is now resolved. Run `npm i nipinit -g` to update now!
If you have any more problems, please report them! 😊 Thanks 🙂
